ClinGen/clincoded
# 2123 PHI Disclaimer Text Change

Related issue: #2123 

### Test steps used by curators

Test instance: https://2123-sc-phi-disclaimer-text-change-rev04.demo.clinicalgenome.org/

- View list of all Variant Interpretations
- Go into a variant whose interpretation hasn't been started
- Click "Interpretation +"
- The modal poped up, verify the text is the same as [mentioned in issue page](https://github.com/ClinGen/clincoded/issues/2123#issue-554970619)
